src/avatar.jsx

```diff
@@ -129,11 +130,11 @@ const Avatar = React.createClass({
       );
     } else {
       styles.root = this.mergeStyles(styles.root, {
-        backgroundColor: backgroundColor,
+        backgroundColor: backgroundColor || this.getTheme().backgroundColor,
         textAlign: 'center',
         lineHeight: size + 'px',
         fontSize: size / 2 + 4,
-        color: color,
+        color: color || this.getTheme().color,
```

Comment:
That's introducing a breaking change.
The doc:
![capture d ecran 2016-01-21 a 14 54 17](https://cloud.githubusercontent.com/assets/3165635/12482246/e1f186f0-c04e-11e5-998b-8ae64173ed36.png)
The spec:
![capture d ecran 2016-01-21 a 14 55 00](https://cloud.githubusercontent.com/assets/3165635/12482261/f3ec46e2-c04e-11e5-83d7-1ee314d38b4a.png)